src/transform/EditTransform.js

```diff
@@ -8,8 +8,17 @@ const CLASS = {
   PROTECTION: { UNPROTECTED: '', PROTECTED: 'page-protected', FORBIDDEN: 'no-editing' }
 }
 
+const IDS = {
+  TITLE_DESCRIPTION: 'pagelib_edit_section_title_description',
+  ADD_TITLE_DESCRIPTION: 'pagelib_edit_section_add_title_description',
+  DIVIDER: 'pagelib_edit_section_divider',
+  PRONUNCIATION: 'pagelib_edit_section_title_pronunciation'
+}
+
 const DATA_ATTRIBUTE = { SECTION_INDEX: 'data-id', ACTION: 'data-action' }
 const ACTION_EDIT_SECTION = 'edit_section'
```

Comment:
Is it possible to have a const for the lead edit section?
 e.g. `const ACTION_EDIT_LEAD_SECTION = 'edit_lead_section'`

The Wikipedia android will popup an overflow menu when clicking on the edit anchor in the lead section. Otherwise will need to override the attribute by scanning the whole HTML.

Reply:
When you detect the click event has 'data-action' of 'edit_section', you can check if 'data-id' is zero to see if the click is for the lead section. Or am I misunderstanding what you're trying to do?

Reply:
Thanks @montehurd!

The reason is that we have another action listener for the edit icon in the lead section. https://github.com/wikimedia/apps-android-wikipedia/blob/master/www/js/editaction.js#L9

It should be ok if stay as it is since we can also update the attribute manually in our `section.js`
https://gerrit.wikimedia.org/r/#/c/apps/android/wikipedia/+/476787/3/www/js/sections.js

Reply:
Ah ok sounds good! I'm looking at your other comments now but I may have to chip away at it over next couple days - also having to do some native work too :)

Reply:
Ya on [line 5](https://github.com/wikimedia/apps-android-wikipedia/blob/master/www/js/editaction.js#L5) it looks like you can just check if 'data-id' is 0 and send the bridge message from [line 10](https://github.com/wikimedia/apps-android-wikipedia/blob/master/www/js/editaction.js#L10).